aiohttp/web.py

```diff
@@ -436,6 +451,7 @@ def run_app(app, *, host=None, port=None, path=None, sock=None,
         servers = loop.run_until_complete(
             asyncio.gather(*server_creations, loop=loop)
         )
+        app['asyncio_servers'] = servers
```

Comment:
You could also simply implement a public attribute like "server" in the instance and make a setter that will update it for you. Just like `set_loop`.

Reply:
IMHO if the idea is just give a way to gather that information, I will go for a simple `getter`. Indeed I would avoid the `setter` taking into account that modifies the content will do nothing.